serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -69,6 +69,8 @@
 ** xref:integrations/expose-metrics-to-prometheus.adoc[Exposing workflow base metrics to Prometheus]
 // ** xref:integrations/camel-k-integration.adoc[Integrating with Camel-K]
  ** xref:integrations/serverless-dashboard-with-runtime-data.adoc[Displaying workflow data in dashboards]
+* Supporting Services
+** xref:supporting-services/jobs-service.adoc[Job Service]
```

Comment:
It would be good to unify naming in the docs. Job/Jobs service.

Reply:
we already brought this to discussion but no decision, like `job services vs jobs service`, I can change it once we define it, no problem.